docs/source/examples.rst

```diff
@@ -426,7 +436,6 @@ For debugging, remember that it is helpful to enable :ref:`Debug and Logging Opt
 
 This example is best for a quick test where you want to check the effect of a
 ping, or where situations where you want to customize when the ping is sent.
-This type of connection does not automatically respond to a "ping" with a "pong".
```

Comment:
At first I was confused because we were discussing ping/pong in the WebSocketApp `run_forever()` case, but a quick test confirms that you are correct about this comment being inaccurate. Good call